dev/aura.html

```diff
@@ -119,7 +119,15 @@
                   { text: 'Move to Project…' },
                   { text: 'Move to Folder…' },
                   { component: 'hr' },
-                  { text: 'Advanced Options…' },
+                  {
+                    text: 'Advanced Options',
+                    children: [
+                      { text: 'Show All', checked: true, className: 'checkable' },
```

Comment:
What about Flow component `setCheckable()` implementation, should it use this new class name?